cpu/lpc2387/Makefile.features

```diff
@@ -1 +1,9 @@
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_gpio periph_gpio_irq
+FEATURES_PROVIDED += periph_pwm
+FEATURES_PROVIDED += periph_rtc
+FEATURES_PROVIDED += periph_spi
+FEATURES_PROVIDED += periph_timer
+FEATURES_PROVIDED += periph_uart
```

Comment:
As I understand it `periph_pwm`, `periph_spi` and `periph_uart` should stay in the board `Makefile.features` as they require board specific configuration.

Reply:
Indeed